superset-frontend/src/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -134,6 +136,19 @@ export default class SqlEditorLeftBar extends React.PureComponent {
     this.props.actions.addTable(this.props.queryEditor, tableName, schemaName);
   }
 
+  renderExpandIconWithTooltip = ({ isActive }) => (
+    <IconTooltip
+      style={{ transform: 'rotate(90deg)' }}
```

Comment:
can we use the css prop here and below instead of `style`? It's a pattern we've been migrating to.

Reply:
Certainly!